src/highlevel_api.jl

```diff
@@ -295,3 +295,20 @@ function _bytes2string(x::Integer)
 end
 
 has_object_of_type(t) = any(obj -> obj.type_ == t, gettopology())
+
+"""
+Shows a graphical visualization of the system topology.
+The quality of the result might depend on the used terminal and might vary between machines and operating systems.
+
+**Note:** The specific visualization may change between minor versions.
+"""
+function topology_graphical()
+    if isdefined(Hwloc_jll, :lstopo_no_graphics)
+        Hwloc_jll.lstopo_no_graphics() do lstopo
+            run(`$lstopo --no-io --no-legend --of txt`)
+        end
+    else
+        error("Requires Hwloc_jll version 2.8.0+1 or higher.")
```

Comment:
We'd at least need 2.8.0+1. Can our compat entries handle the +1 part? I.e. does `Hwloc_jll = "2.8.0+1" work?

Reply:
You can't because Pkg doesn't distinguish between different build numbers, but it always uses the latest one: asking for `2.8.0` effectively means `2.8.0+1` (or whatever next build there will be)

Reply:
Also, the Project.toml still has `julia = 1.3` whereas Hwloc_jll has `julia = 1.6`. But I guess I should just bump the former to 1.6 as well. (We also don't test anything before 1.6 anymore.)

Reply:
Oh yeah, also the syntax I suggested in https://github.com/JuliaParallel/Hwloc.jl/pull/62#discussion_r979893515 is only for v1.6+ :grimacing: